galleon-pack/src/main/resources/configs/standalone/standalone-microprofile-ha.xml/config.xml

```diff
@@ -7,10 +7,12 @@
         <include name="microprofile-fault-tolerance"/>
         <include name="microprofile-jwt"/>
         <include name="microprofile-openapi"/>
+        <include name="microprofile-opentracing"/>
         <include name="undertow-legacy-https"/>
         <exclude name="management-security-realm"/>
         <exclude name="jpa"/>
         <include name="jpa-distributed"/>
         <include name="web-clustering"/>
     </layers>
+    <feature-group name="microprofile-opentracing-jaeger"/>
```

Comment:
The current standalone-microprofile-ha.xml uses the empty config.

Reply:
ok, I've realigned the produced configurations with the ones in 21